R/run_mcmc.R

```diff
@@ -110,14 +110,37 @@ run_mcmc_p <- function(dt, priorObj, n.chains, n.adapt, n.burn, n.iter, seed, pa
     cat("seed_list:", seed_list, "number of dataset", length(dt), "\n")
   )
 
-  message(n.cores, " clusters are being used")
+  ps_message(n.cores, " clusters are being used")
   cl <- parallel::makeCluster(n.cores)
   doParallel::registerDoParallel(cl)
 
+  # Load psborrow on the clusters
+  ## Function to see whether psborrow is being used in a development context
+  is_psborrow_dev <- function() {
+    desc <- file.path("DESCRIPTION")
+    if (file.exists(desc)) {
+      content <- read.dcf(desc)
+      pkg <- content[1, ][["Package"]]
+      return(pkg == "psborrow")
+    }
+    return(FALSE)
+  }
```

Comment:
This should be defined as separate function, there isn't any reason for it to be a closure.